### DISCUSSION TITLE
Is this a licence violation?

### DISCUSSION THREAD

**Discussion by ziqiangao** [TurboWarp/extensions#1296] [Q&A] [answered]
I am trying to make a fluidsynth extension which depends on two libraries,
[fluidsynth-emscripten](https://github.com/jet2jet/fluidsynth-emscripten) which has the following [LGPL-2.1 license](https://github.com/jet2jet/fluidsynth-emscripten?tab=LGPL-2.1-1-ov-file#)

[js-synthesizer](https://github.com/jet2jet/js-synthesizer) which has the following [BSD-3-Clause license](https://github.com/jet2jet/js-synthesizer?tab=BSD-3-Clause-1-ov-file#)

Are these two licences compatible with [GNU General Public License version 3](https://github.com/ziqiangao/extensions/blob/master/licenses/GPL-3.0.txt)

**Comment by GarboMuffin** [comment, marked as answer]
those should be okay, just plop them into `licenses` if they aren't already in there and leave appropriate comments

i am sorry about the delay, everything is very slow, your choice of library was not the reason for the delay
